packages/core/src/components/Card/styles.ts

```diff
@@ -10,6 +10,14 @@ export const styleSheetCard: StyleSheet = ({ color, pattern }) => ({
   card_overflow: {
     overflow: 'visible',
   },
+
+  card_noShadow: {
+    border: ui.border,
+  },
+
+  card_selected: {
+    border: `2px solid ${color.core.primary[3]}`,
```

Comment:
can you use [`ui.borderThick`](https://github.com/airbnb/lunar/blob/master/packages/core/src/themes/buildTheme.ts#L56) here instead?

Reply:
Sure, will do.